scripts/embedder.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
```

Comment:
I like it. More control on the header is good. I'll try something later. Thanks!

Reply:
Sorry, I've only had time to try the hex representation. Cool because it's more compact than decimal BUT now as it's a string we have offended the compiler:

`warning: string literal of length 6998518 exceeds maximum length 65536 that C++ compilers are required to support [-Woverlength-strings]`

Not sexy anymore. The array doesn't have that limitation, and after all it lives as a fat source only for a brief lapse until it's compiled to object. 

By the way @mooskagh your int to hex had issues. I'm not a python guy but `bytes()` is [0..255] and it wasn't needed, worked with just `rb'\x%02x' % x`.